### PR TITLE
Databricks Schema Browser: Allow eventlet worker instead of RQ

### PR DESCRIPTION
**Pull request by gabrieldutra**
## What type of PR is this? (check all applicable)

- [x] Other

## Description
Use eventlet worker in order to make the Databricks Schema Browser more responsive.

## Related Tickets & Documents
--

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
--

### CODE REVIEW

File: bin/docker-entrypoint [hunk 1]
```diff
@@ -32,7 +32,7 @@ server() {
   # Recycle gunicorn workers every n-th request. See http://docs.gunicorn.org/en/stable/settings.html#max-requests for more details.
   MAX_REQUESTS=${MAX_REQUESTS:-1000}
   MAX_REQUESTS_JITTER=${MAX_REQUESTS_JITTER:-100}
-  exec /usr/local/bin/gunicorn -b 0.0.0.0:5000 --name redash -w${REDASH_WEB_WORKERS:-4} redash.wsgi:app --max-requests $MAX_REQUESTS --max-requests-jitter $MAX_REQUESTS_JITTER
+  exec /usr/local/bin/gunicorn -b 0.0.0.0:5000 --name redash -w${REDASH_WEB_WORKERS:-4} redash.wsgi:app --max-requests $MAX_REQUESTS --max-requests-jitter $MAX_REQUESTS_JITTER --worker-class eventlet
```
Comment (arikfr) [inline bin/docker-entrypoint:35]:
```suggestion
  exec /usr/local/bin/gunicorn -b 0.0.0.0:5000 --name redash -w${REDASH_WEB_WORKERS:-4} redash.wsgi:app --max-requests $MAX_REQUESTS --max-requests-jitter $MAX_REQUESTS_JITTER
```

Let's not enable this by default until we know it's "good". We can enable it in our setup using the `GUNICORN_CMD_ARGS` env variable.

File: client/app/services/databricks-data-source.js [hunk 2]
```diff
@@ -5,12 +5,24 @@ import { fetchDataFromJob } from "@/services/query-result";
...
-      .then(data => (has(data, "job.id") ? fetchDataFromJob(data.job.id, 300).catch(() => []) : Promise.resolve([]))),
+  getDatabases: ({ id }, refresh = false) => {
+    const params = {};
+
+    if (refresh) {
+      params.refresh = true;
+    }
+    return axios
+      .get(`api/databricks/databases/${id}`, { params })
+      .then(data => (has(data, "job.id") ? fetchDataFromJob(data.job.id, 300).catch(() => []) : data));
```
Comment (arikfr) [inline client/app/services/databricks-data-source.js:16]: We still call `fetchDataFromJob` to make it easier to switch to RQ in the future?

Comment (gabrieldutra) [reply client/app/services/databricks-data-source.js:16]: The idea was on those lines, but I can actually remove it for now :)

File: .circleci/config.yml [hunk 3]
```diff
@@ -108,6 +108,11 @@ jobs:
       - run:
           name: Execute Cypress tests
           command: npm run cypress run-ci
+      - run:
+          name: "Failure: output container logs to console"
+          command: |
+            docker-compose logs
+          when: on_fail
```
Comment (gabrieldutra) [inline .circleci/config.yml:115]: This part is because I had to debug when it failed. It's good to keep anyway.

**Comment by arikfr** [comment]
👍

**Pull request merged**